ocs_ci/ocs/resources/mcg.py

```diff
@@ -883,21 +888,25 @@ def _compare_cli_hashes():
     @property
     def status(self):
         """
-        Verify noobaa status output is clean without any errors
+        Verify the status of NooBaa, and its default backing store and bucket class
 
         Returns:
-            bool: return False if any of the non optional components of noobaa is not available
+            bool: return False if any of the above components of noobaa is not in READY state
 
         """
         # Get noobaa status
-        status = self.exec_mcg_cmd("status").stderr
-        for line in status.split("\n"):
-            if (
-                any(i in line for i in ["Not Found", "Waiting for phase ready ..."])
-                and "Optional" not in line
-            ):
-                logger.error(f"Error in noobaa status output- {line}")
-                # Workaround for https://github.com/red-hat-storage/ocs-ci/issues/3570
-                return True
-        logger.info("Verified: noobaa status does not contain any error.")
-        return True
+        get_noobaa = OCP(kind="noobaa", namespace=self.namespace).get(
+            resource_name="noobaa"
+        )
+        get_default_bs = OCP(kind="backingstore", namespace=self.namespace).get(
+            resource_name=DEFAULT_NOOBAA_BACKINGSTORE
+        )
+        get_default_bc = OCP(kind="bucketclass", namespace=self.namespace).get(
+            resource_name=DEFAULT_NOOBAA_BUCKETCLASS
+        )
+        return (
+            get_noobaa["status"]["phase"]
```

Comment:
we have seen that in some cases, `["status"]["phase"]` does not return an up to date status. 
I suggest we use ocs_ci.ocs.ocp.OCP.get_resource_status instead

Reply:
we cant use this function since the get on noobaa resources does not return a status column, it returns a phase column instead

Reply:
so I think you can use ocs_ci.ocs.ocp.OCP.get_resource and pass `phase` column instead of `status`

Reply:
discussed offline.
The concern is because of some other resources, such as node, which their phase status does not change when it needs to change.
We just need to make sure that for `noobaa`, `backingstore` and `bucketclass` resources, `["status"]["phase"]` gets updated as it should